templates/node-class.mustache

```diff
@@ -14,6 +14,15 @@ var {{&className}} = (function(){
         this.domain = domain;
     }
 
+    {{#isSecure}}
```

Comment:
@borisirota Do you think it could be possible to set the token in the constructor?